### PR TITLE
Adding Unverified List (UVL) data into Watchman search

### PR DESCRIPTION
**Pull request by ckelly-digicert**
### Adding Unverified List (UVL) data into Watchman
- Updated model to process UVL data from the core US Sanctions CSV

### CODE REVIEW

**Comment by ckelly-digicert** [comment]
Do note that I'm still working on the unit tests. Please treat this PR as a draft until these tests have been brought in.

**Comment by adamdecaf** [comment]
@ckelly-digicert I pushed a fix to the download metrics. You'll want to pull the ammended commit. 

I can view results with curl, so I'm not sure what issue you were seeing. Running `make` and then `go run ./cmd/server`  let's us query the UVL list. 

```
$ curl -s "http://localhost:8084/search?q=Alfa+Photonics&limit=1" | jq .unverified 
[
  {
    "Addresses": [
      "21 Krisjana Valdemara Iela, Riga, Latvia, LV",
      "151–11 Krisjana Valdemara Iela, Riga, Latvia, LV",
      "52–66 Darza Iela, Riga, Latvia, LV",
      "Nordic Technology Park, 15/25 Jurkalnes Iela, Riga, Latvia, LV"
    ],
    "EntityID": "",
    "Name": "Alfa Photonics",
    "SourceInfoURL": "http://bit.ly/1Qi4R7Z",
    "SourceListURL": "http://bit.ly/1iwwTSJ",
    "match": 1
  }
]
```

File: cmd/server/download.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ type DownloadStats struct {
 	BISEntities       int `json:"bisEntities"`
 	MilitaryEndUsers  int `json:"militaryEndUsers"`
 	SectoralSanctions int `json:"sectoralSanctions"`
+	Unverified        int `json:"Unverified"`
```
Comment (adamdecaf) [inline cmd/server/download.go:65]: Just to be clear where these entries came from. 

```suggestion
	Unverified        int `json:"unverifiedCSL"`
```

File: cmd/server/search_handlers.go [hunk 2]
```diff
@@ -158,6 +158,7 @@ type searchResponse struct {
 	BISEntities       []*Result[csl.EL]  `json:"bisEntities"`
 	MilitaryEndUsers  []*Result[csl.MEU] `json:"militaryEndUsers"`
 	SectoralSanctions []*Result[csl.SSI] `json:"sectoralSanctions"`
+	Unverified        []*Result[csl.UVL] `json:"unverified"`
```
Comment (adamdecaf) [inline cmd/server/search_handlers.go:161]:
```suggestion
	Unverified        []*Result[csl.UVL] `json:"unverifiedCSL"`
```

**Review by adamdecaf — COMMENTED** [review]
This is looking great. Just a tweak for the JSON should help clarify.

**Comment by adamdecaf** [comment]
I pushed https://github.com/moov-io/watchman/commit/fc40f487b30a8e5d4f5d4d424fdcee7e48957fed with fixes for go1.19's `gofmt` change. Can you rebase off `origin/master`?

**Comment by adamdecaf** [comment]
Ah yea there were more deprecations that didn't show up at first. I've fixed them in https://github.com/moov-io/watchman/commit/52dd4fff3cc54362dface613141680f0058de1c8

**Review by adamdecaf — APPROVED** [review]

**Pull request merged**